cloudfront.tf

```diff
@@ -1,9 +1,5 @@
-# Since Terraform not support conditional in the resources atm
-# https://github.com/hashicorp/terraform/issues/15974
-#   => we need to duplicate "aws_cloudfront_distribution" to support "lambda@edge" feature
-
 resource "aws_cloudfront_distribution" "main" {
-  count = var.lambda_edge_enabled ? 0 : 1
+  count = 1
```

Comment:
Since we use new v12 syntax to create lambda blocks, `count` no need here. We can consider to remove this later, (big statefile update for legacy users).

Reply:
We can remove this if it's not affecting users' actual cloud state.

Reply:
so we can not remove it since the resource will be replaced (tested on my local)